client/my-sites/earn/home.tsx

```diff
@@ -136,11 +134,16 @@ const Home: FunctionComponent< ConnectedProps > = ( {
 						'Accept credit card payments via PayPal for physical products, digital goods, services, donations, or support of your creative work.'
 				  )
 				: translate(
-						'Accept credit card payments via PayPal for physical products, digital goods, services, donations, or support of your creative work. {{em}}Available with a Premium, Business, or eCommerce plan{{/em}}.',
+						'Accept credit card payments via PayPal for physical products, digital goods, services, donations, or support of your creative work. {{em}}Available with a %(planNames)s plan{{/em}}.',
 						{
 							components: {
 								em: <em />,
 							},
+							args: {
+								planNames: isJetpackNotAtomic
+									? translate( 'Premium or Professional', { context: 'plan names' } )
```

Comment:
:information_source: This string doesn't exist elsewhere without a **context** (or with a different one). You can probably drop the context. If needed, you can use a translator comment instead to clarify meaning.

Reply:
I think this makes sense and we should use `comment` rather than `context`.

```suggestion
									? translate( 'Premium or Professional', { comment: 'plan names' } )
```

Further reading: https://wpcalypso.wordpress.com/devdocs/packages/i18n-calypso/README.md#options